src/index.ts

```diff
@@ -170,52 +171,6 @@ export default async function (executionContext: FExecutionContext, settings: Se
 				await httpPublisherInstance.init(executionContext);
 				itemsToDispose.push(httpPublisherInstance);
 			}
-
-			// Setup egresses
```

Comment:
Not sure... Why to you remove Egress setup? (where is it moved?)

Reply:
Restore.